indigo/analysis/refs/base.py

```diff
@@ -157,31 +160,22 @@ def setup(self, root):
         self.ancestor_xpath = etree.XPath('|'.join(f'.//a:{a}' for a in self.ancestors), namespaces=self.nsmap)
         self.candidate_xpath = etree.XPath(self.candidate_xpath, namespaces=self.nsmap)
 
-    def make_href(self, node, match):
```

Comment:
Moved this into subclass since its tradition specific (knowing to find "section" elements).